Add note about /p arguments (#23997)
dotnet/docs: docs/core/tools/dotnet-build.md

In addition to its options, the `dotnet build` command accepts MSBuild options, such as `-p` for setting properties or `-l` to define a logger. For more information about these options, see the [MSBuild Command-Line Reference](/visualstudio/msbuild/msbuild-command-line-reference). Or you can also use the [dotnet msbuild](dotnet-msbuild.md) command.

> [!NOTE]

> > When `dotnet build` is run automatically by `dotnet run`, arguments like `/property:property=value` are not respected.

Reviewer: nit

```suggestion
> When `dotnet build` is run automatically by `dotnet run`, arguments like `/property:property=value` aren't respected.
```
